data/compatible_products.yml

```diff
@@ -121,6 +142,7 @@ sets:
         versions:
         - version: 5.1.0
           compatibility: 'https://eclipse-ee4j.github.io/glassfish/certifications/jakarta-web-profile/8.0/TCK-Results'
+          version_download: 'https://eclipse-ee4j.github.io/glassfish/certifications/jakarta-web-profile/8.0/TCK-Results'
```

Comment:
"version_download" seems redundant here. Can we rename this to "download_url" instead of "version_download"?

Reply:
renamed